VOLTA/data/entsoe/hdfwriter.py

```diff
@@ -84,6 +85,48 @@ def get_bidding_zones_with_data(self) -> None:
             if csv_path.exists():
                 self.bidding_zones_with_data.append(zone)
 
+    def get_pricing_from_csv(self, bidding_zone: str) -> pd.DataFrame:
+        """
+        Get pricing from CSV files for a given bidding zone.
+
+        Parameters
+        ----------
+        bidding_zone : str
+            The bidding zone.
+
+        Returns
+        -------
+        pd.DataFrame
+            A dataframe with pricing data for all time intervals for the requested bidding zone.
+        """
+        log.info(f"Reading pricing from CSV file for bidding zone {bidding_zone}.")
+        dfs_per_year = []
+        csv_path = self.csv_path / bidding_zone
+        log.info(f"CSV base path is {csv_path}.")
+
+        for idx, date in enumerate(self.intervals[:-1]):
+            year = date[:4]  # Extract year from date.
+            log.debug(f"In year {year}...")
+            path_to_pricing_file = (
+                csv_path / year / f"pricing_{bidding_zone}_{year}.csv"
+            )
+            index = pd.date_range(
+                year, self.intervals[idx + 1], freq=self.frequency, tz="UTC"
+            )
+            if path_to_pricing_file.is_file():
+                pricing = pd.read_csv(path_to_pricing_file, index_col=0)
+                pricing.index = pd.to_datetime(pricing.index, utc=True)
+                pricing = pricing.reindex(index)
+            else:
+                pricing = pd.DataFrame(np.nan, index=index, columns=["price"])
+                log.debug("No file present, interpret data as Nan")
```

Comment:
```suggestion
                log.debug("No file present, interpret data as NaN.")
```
Sorry for this one 🙈. I kind of have an eye for random, not so important details like this. However, consistency is always nice 😌.